src/main/java/buildcraftAdditions/blocks/BlockBackpackStand.java

```diff
@@ -16,22 +17,21 @@
 public class BlockBackpackStand extends BlockBase {
 
 	public BlockBackpackStand() {
-		super();
-		setBlockName("BlockBackpackStand");
+		super("backpackStand");
 	}
 
 	@Override
-	public void onBlockPlacedBy(World p_149689_1_, int p_149689_2_, int p_149689_3_, int p_149689_4_, EntityLivingBase p_149689_5_, ItemStack p_149689_6_) {
-		super.onBlockPlacedBy(p_149689_1_, p_149689_2_, p_149689_3_, p_149689_4_, p_149689_5_, p_149689_6_);
+	public void onBlockPlacedBy(World world, int x, int y, int z, EntityLivingBase entity, ItemStack stack) {
```

Comment:
same as the kinetic backpack, work in progress, was already changed locally but not pushed, this was just to test what functions returned what